client/config.xml

```diff
@@ -39,6 +39,10 @@
 
         <preference name="AndroidWindowSplashScreenAnimatedIcon" value="resources/splashscreen/android/splash_screen_icon_xxxhdpi.png" />
         <preference name="AndroidWindowSplashScreenBackground" value="#263237" />
+        <preference name="SplashScreenDelay" value="3000" />
```

Comment:
This feels like a really long time. What does it look like if we make this zero?

Reply:
With no delay it jumps around some pre-rendering screens in a very choppy way.

We can tune the value (3000 actually seems too short on some slower-loading setups) but 0 is definitely not correct.

[Screen_recording_20250715_182852.webm](https://github.com/user-attachments/assets/20c4bbf6-723f-4a1b-845e-974f9398c400)

Reply:
Can we have code in main() that dismisses the splashscreen?

Reply:
@daniellacosse are we setting the theme after the page load?